Drivers/Src/Pedals.c

```diff
@@ -37,26 +37,28 @@ void PedalsInit() { BspAdcInit(); }
  * @param   Pedal, ACCELERATOR or BRAKE as defined in enum
  * @return  percent amount the pedal has been pressed in percentage
  */
-int8_t PedalsRead(Pedal pedal) {
+uint8_t PedalsRead(Pedal pedal) {
     if (pedal >= kNumberOfPedals) {
         return 0;
     }
-    int16_t millivolts_pedal =
-        (int16_t)BspAdcGetMillivoltage((pedal == kAccelerator) ? kCh10 : kCh11);
-
-    int8_t percentage = 0;
 
-    if (millivolts_pedal >= kLowerBound[pedal]) {
-        percentage = (int8_t)((int32_t)(millivolts_pedal - kLowerBound[pedal]) *
-                              100 / (kUpperBound[pedal] - kLowerBound[pedal]));
-    }
+    int16_t millivolts_pedal =
+        BspAdcGetMillivoltage((pedal == kAccelerator) ? kCh10 : kCh11);
```

Comment:
Can you also make this function return an unsigned int? Take a look at BSP_ADC.c

Reply:
Sounds good - it looks like both the getters should be using unsigned ints so I'll change them both.